cvat/apps/engine/data_manager.py

```diff
@@ -133,6 +133,9 @@ def merge(self, objects, start_frame, overlap):
                     if j != -1:
                         self._modify_unmached_object(old_objects[j],
                             start_frame + overlap)
+                        # Need to reset cached interpolated shapes
+                        # because shapes was changed 
+                        old_objects[j]['interpolated_shapes'] = []
```

Comment:
Should we have the condition inside _modify_unmached_object? Or it isn't a good solution by a reason?